.github/workflows/push-image.yml

```diff
@@ -3,184 +3,126 @@ name: Push Stack Image
 on:
   release:
     types:
-    - published
+      - published
```

Comment:
@pacostas just confirmining this change was on purpose ?

Reply:
I formatted the document with prettier (for faster development) and I also see that github uses two spaces on the attributes under the types e.x. [https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows . ](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-pull_request-workflow-based-on-files-changed-in-a-pull-request) so prettier probably is aligned to what github suggests

Reply:
It would be nice to have a standard way of formatting yml files as I see there is not a consistent way across yml files. e.x. this one https://github.com/paketo-buildpacks/github-config/blob/main/stack/.github/workflows/push-image.yml vs this one https://github.com/paketo-buildpacks/github-config/blob/main/stack/.github/workflows/synchronize-labels.yml

Reply:
@pacostas if thre is no linter/formatter used in any of the other Paketo repositories maybe you can propose one, as I think that would be a good idea.